scripts/twister

```diff
@@ -44,6 +44,10 @@ pairs:
     Extra configuration options to be merged with a master prj.conf
     when building or running the test case.
 
+  sysbuild: <True|False> (default False)
+    If true, build the sample using the sysbuild infrastructure. This
+    will disable filtering based on Kconfig or devicetree information.
```

Comment:
if this is just temporary in order to limit the scope, then fine, but long term we should be able to still allow Kconfig or dt filtering and just use the `.config` or dt from the main sample build folder instead.

But I can accept the limitation for now.

Reply:
I need to update this, thank you for pointing it out. KConfig and devicetree filtering are supported, just only on the main systembuild application

Reply:
This has been updated to reflect the actual support present in this PR